### PR TITLE
Add test case for svgs inlined in text and add accessibility text for other svg tests

### PR DESCRIPTION
**Pull request by warren-ms**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [ ] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes
Adds a test for svgs embedded in a text run
Add accessibility properties for screen readers for svgs used in tests. This is from https://github.com/microsoft/fluentui-react-native/pull/3618 but the PR builds seemed to get stuck in a un-rerunnable state

### Verification

(how the change was tested, including both manual and automated tests)

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### Pull request checklist

This PR has considered (when applicable):
- [x] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

File: apps/fluent-tester/src/TestComponents/Svg/SvgTest.tsx [hunk 1]
```diff
@@ -58,41 +59,81 @@ const RectTest: React.FunctionComponent = () => {
...
+  const toggleShowInline = () => {
+    setShowInline(!showInline);
+  };
+  const toggleRectFill = () => {
+    setIsGreen(!isGreen);
+  };
+
+  return (
+    <>
+      <TextV1 style={{ textTransform: 'uppercase' }} font="base" weight="medium" truncate>
```
Comment (rurikoaraki) [inline apps/fluent-tester/src/TestComponents/Svg/SvgTest.tsx:75]: style as const object please

File: apps/fluent-tester/src/TestComponents/Svg/SvgTest.tsx [hunk 2]
```diff
@@ -58,41 +59,81 @@ const RectTest: React.FunctionComponent = () => {
   );
 };
 
+const RichTextInlineSvgTest: React.FunctionComponent = () => {
+  const [showInline, setShowInline] = React.useState(false);
+  const [isGreen, setIsGreen] = React.useState(true);
+
+  const toggleShowInline = () => {
```
Comment (rurikoaraki) [inline apps/fluent-tester/src/TestComponents/Svg/SvgTest.tsx:66]: Should use React.useCallback

same as below

File: apps/fluent-tester/src/TestComponents/Svg/SvgTest.tsx (hunk 1 above)
Comment (warren-ms) [reply apps/fluent-tester/src/TestComponents/Svg/SvgTest.tsx]: Moved into const obj

File: apps/fluent-tester/src/TestComponents/Svg/SvgTest.tsx (hunk 2 above)
Comment (warren-ms) [reply apps/fluent-tester/src/TestComponents/Svg/SvgTest.tsx]: Switched to useCallback

**Review by rurikoaraki — APPROVED** [review]

**Pull request merged**